citron-collection-partitioner/README.md

```diff
@@ -0,0 +1,16 @@
+# Data Stack Scrapper
+Python script used to partition mongo collections data stored on aws s3 bucket.
```

Comment:
Python script used to partition mongo collections data stored on aws s3 bucket.

-->

Python script used to test AWS Athena partition strategy performances for data stored on S3.